vendor/diff-parsers-upstream-std.patch

```diff
@@ -132,6 +132,36 @@
    | '#' symbolchar_or_hash + as op
              { HASHOP op }
    | "let" kwdopchar dotsymbolchar * as op
+@@@@
+   | (_ as illegal_char)
+       { error lexbuf (Illegal_character illegal_char) }
+ 
+ and directive = parse
+   | ([' ' '\t']* (['0'-'9']+ as num) [' ' '\t']*
+-        ("\"" ([^ '\010' '\013' '\"' ] * as name) "\"") as directive)
++        ("\"" ([^ '\010' '\013' '\"' ] * as _name) "\"") as directive)
+         [^ '\010' '\013'] *
+       {
+         match int_of_string num with
+         | exception _ ->
+             (* PR#7165 *)
+             let explanation = "line number out of range" in
+             error lexbuf (Invalid_directive ("#" ^ directive, Some explanation))
```

Comment:
Why try to decode the line number at all ? The whole match can be removed.

Reply:
done